Week1/exercise/w1/index.js

```diff
@@ -15,7 +10,28 @@ console.log('header: ', header);
 //    set image src to imageInput element value
 
 // ====================================== //
+const ChangeImageButton = document.querySelector('#btn-changeImage');
+console.log(ChangeImageButton);
+const changeImage = () => {
+  const imgElement = document.querySelector('#imageToChange');
```

Comment:
better to define these variables outside this function such that they are only selected once and not every time this function is called